ajout du header file display.h

philouxy opened the pull request:
fichier de type header file pour toutes les fonctions liées à l'afficahge

philouxy committed:
Update README.md

philouxy committed:
Create main_langage_C_fixme

philouxy committed:
Create main_langage_C_fixme.c

philouxy committed:
Create display.h

philouxy committed:
Update main_langage_C_fixme.c

philouxy committed:
Create template_menu.c

philouxy committed:
Merge pull request #1 from philouxy/code

Create main_langage_C_fixme

gcmalloc commented on main_langage_C_fixme:
```diff
@@ -0,0 +1,46 @@
+// Nom du projet 		  :	    Langage_C_fixme
+// Nom du fichier 		:     main_langage_C_fixme.c
+// Date de création 	:     23.02.2015
+// Date de modification : 	xx.xx.2015
```
align the :

philouxy replied:
Yes, you are right. in my text editor is correct, but not here... I will place correct the indentation

gcmalloc commented on main_langage_C_fixme.c:
```diff
@@ -0,0 +1,55 @@
+    printf("3/ ... \n ");
+    printf("4/ ... \n ");
+    printf("--- help ---");
+    SAUT_LIGNE;
```
PRINT_SAUT_LINGNE is even clearer, for me SAUT_LIGNE is '\n'

philouxy committed:
Update display.h

philouxy committed:
Update template_menu.c

philouxy committed:
Update main_langage_C_fixme.c

philouxy committed:
Delete main_langage_C_fixme

philouxy committed:
Update main_langage_C_fixme.c

philouxy committed:
Update display.h

philouxy committed:
Update template_menu.c

philouxy committed:
Update display.h

add some #define for the c file template menu

philouxy committed:
Update main_langage_C_fixme.c

add of loop for the selection of menu

philouxy committed:
Update template_menu.c

add the Template_Choix_Menu_Arithmetique function and modification of Template_Choix_Menu function

philouxy committed:
Update main_langage_C_fixme.c

Implement the user selection : getchar();... it allows to not turn the loop endlessly

philouxy committed:
Update display.h

change name on "Template_Menu_Ligne" + addition of constants

philouxy committed:
Update main_langage_C_fixme.c

change of structure in the loop "do while"

philouxy committed:
Update template_menu.c

change function : -  Template_Menu_Ligne / Template_Display_Menu / Template_Choix_Menu_Principal

philouxy committed:
Create arithmetique.h

philouxy committed:
Create calcul_entier.c
